src/components/map/dynamicMap.tsx

```diff
@@ -8,38 +8,36 @@ import {
   MapLayerMouseEvent,
   ViewStateChangeEvent,
   Popup,
-  Layer,
   LngLatBoundsLike,
 } from "react-map-gl/maplibre";
 import maplibregl, {
   FilterSpecification,
   LayerSpecification,
   LineLayerSpecification,
+  GeoJSONSource,
 } from "maplibre-gl";
 import { Protocol } from "pmtiles";
 import { sources } from "./helper/sources";
 import { AppDispatch, RootState } from "@/store";
-import { setBboxParam, setVisLyrs } from "@/store/slices/searchSlice";
+import { setBboxParam } from "@/store/slices/searchSlice";
 import "maplibre-gl/dist/maplibre-gl.css";
 import { overlayRegistry, layerRegistry } from "./helper/layers";
-import DynamicMapButtons from "./dynamicMapButtons";
+
+import "@maptiler/geocoding-control/style.css";
+
+import * as turf from "@turf/turf";
+
+import GeoSearchControl from "./geoSearchControl";
+
+const apiKey = "bnAOhGDLHGeqBRkYSg8l";
```

Comment:
Actually, I'll add one more commit to make this change right now, why not...

Reply:
Awesome, this key is already in the Git history, so it will be hard to fully remove. As long as we use a different key in .env and ensure it is the one in use, it should be fine. (probably remove or commenting out this line since it is not being used)

Reply:
Yeah, it's revoked now anyway. All good.